src/uucore/src/lib/features/format/num_format.rs

```diff
@@ -326,6 +340,30 @@ impl Formatter for Float {
     }
 }
 
```

Comment:
what about something like 
```
fn get_flag<T: PartialOrd + Default>(sign: PositiveSign, x: T) -> String {
    if x >= T::default() {
        match sign {
            PositiveSign::None => String::new(),
            PositiveSign::Plus => String::from("+"),
            PositiveSign::Space => String::from(" "),
        }
    } else {
        String::from("-")
    }
}
```

Reply:
Nice one! I didn't know about that. Now updated.